### PR TITLE
Optimized getviewsize

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

https://github.com/tgstation/TerraGov-Marine-Corps/pull/5438
## Why It's Good For The Game

image
image

## Changelog
:cl:
code: Optimizes getviewsize()
/:cl:

### CODE REVIEW

File: code/__HELPERS/view.dm [hunk 1]
```diff
@@ -1,15 +1,10 @@
 /proc/getviewsize(view)
-	var/viewX
-	var/viewY
 	if(isnum(view))
-		var/totalviewrange = (view < 0 ? -1 : 1) + 2 * view
-		viewX = totalviewrange
-		viewY = totalviewrange
+		var/totalviewrange = 1 + 2 * view
```
Comment (LemonInTheDark) [inline code/__HELPERS/view.dm:3]: The ternary statement was here for a reason. I don't remember what reason, but there was one

Comment (TiviPlus) [reply code/__HELPERS/view.dm:3]: copypasta will fix

Comment (TiviPlus) [reply code/__HELPERS/view.dm]: Done

**Comment by LemonInTheDark** [comment]
Honestly this is somewhat of a micro op, but it's not a big deal so fuck it

**Review by LemonInTheDark — APPROVED** [review]

**Pull request merged**